core/src/test/java/com/google/zxing/qrcode/decoder/DecodedBitStreamParserTestCase.java

```diff
@@ -81,6 +81,19 @@ public void testHanzi() throws Exception {
     assertEquals("\u963f", result);
   }
 
+  @Test
+  public void testHanziLevel1() throws Exception {
+    BitSourceBuilder builder = new BitSourceBuilder();
+    builder.write(0x0D, 4); // Hanzi mode
+    builder.write(0x01, 4); // Subset 1 = GB2312 encoding
+    builder.write(0x01, 8); // 1 characters
+    // A5A2 (ア, U+30A2) => A5A2 - A1A1 = 401, 4*60 + 01 = 0181
```

Comment:
I'm not sure if it is a problem now but I have historically avoided Unicode in source files. Is this one necessary?

Reply:
It is just a comment. I remove it.